internal/events/get.go

```diff
@@ -55,7 +56,9 @@ func get(
 		end = args[2]
 	}
 
-	events, err := services.Events.Get(name, start, end)
+
+	eventNames := strings.Split(name, ",")
+	events, err := services.Events.GetMany(eventNames, start, end, 249, 10)
```

Comment:
obviously this is not going to be here for the final version, but I added it here to test it out.

Reply:
Not really sure if we want to expose the worker count either. It could be calculated from the queries slices and some properties.